tests/integration-tests/tests/performance_tests/test_osu/test_osu/pcluster.config.yaml

```diff
@@ -0,0 +1,37 @@
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: {{ head_node_instance }}
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+  Imds:
+    Secured: {{ imds_secured }}
+Scheduling:
+  Scheduler: {{ scheduler }}
+  SlurmQueues:
+    - Name: efa-enabled
+      Networking:
+        PlacementGroup:
+          Enabled: true
+          Name: c5n_capacity_reservation
```

Comment:
Another reason to hard wire also the compute nodes instance type: we are hard wiring the c5n PG (and it is correct), so it makes sense to also hard wire the c5n compute capacity rather than using a variable.